tests/k8s/test_k8s.py

```diff
@@ -1120,6 +1120,7 @@ async def test_flow_with_stateful_executor(
         )
 
         core_client.delete_namespace(namespace)
```

Comment:
The error is calling `delete namespace`, what we want is to restart the Pod, so we should delete the Pods